src/components/public/about-us-body.tsx

```
@@ -0,0 +1,109 @@
+import { ABOUT_DEVELOPERS } from '@/constants';
+
+function AboutUsBody() {
+  return (
+    <div className="min-h-screen bg-gradient-to-b from-gray-50 to-gray-100 px-4 py-16 sm:px-6 lg:px-8">
+      <div className="mx-auto max-w-4xl">   
+        <div className="overflow-hidden rounded-2xl bg-white transition-all">
+          <div className="p-8 sm:p-12">
+            <div className="mb-12">
+              <h2 className="mb-6 flex items-center text-2xl font-bold text-gray-800">
+                <span className="mr-4 rounded-lg bg-blue-100 p-2 text-blue-600">
+                  <svg xmlns="http://www.w3.org/2000/svg" className="size-6" fill="none" viewBox="0 0 24 24" stroke="currentColor">
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)